### PR TITLE
Add NSM Dashboard deployment files.

### PR DESCRIPTION
**Pull request by VitalyGushin**
## Description
Add NSM Dashboard deployment files.

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [x] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [x] New functionality
- [x] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: examples/observability/dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Cleanup:
+
+To remove the dashboard from the cluster, execute the command:
+
+```bash
+pkill -f "kubectl port-forward -n nsm-system service/dashboard-backend 3001:3001"
+pkill -f "kubectl port-forward -n nsm-system service/dashboard-ui 3000:3000"
+kubectl delete service/dashboard-ui service/dashboard-backend pod/dashboard -n=nsm-system
+```bash
```
Comment (denis-tingaikin) [inline examples/observability/dashboard/README.md:46]:
```suggestion
```
```

File: examples/observability/dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Run
+
+To run the dashboard in the cluster, execute the command:
+
+```bash
+kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/apps/dashboard
+kubectl wait --for=condition=ready pod -l app=dashboard --timeout=300s -n nsm-system
+nohup kubectl port-forward -n nsm-system service/dashboard-backend 3001:3001 &
+nohup kubectl port-forward -n nsm-system service/dashboard-ui 3000:3000 &
```
Comment (denis-tingaikin) [inline examples/observability/dashboard/README.md:21]: TODO: Run commands separatly

File: apps/dashboard/dashboard-pod.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+metadata:
+  name: dashboard
+  namespace: nsm-system
+  labels:
+    app: dashboard
+    spiffe.io/spiffe-id: "true"
+spec:
+  containers:
+  - name: dashboard-backend
+    image: ghcr.io/networkservicemesh/ci/cmd-dashboard-backend
```
Comment (denis-tingaikin) [inline apps/dashboard/dashboard-pod.yaml:13]:
```suggestion
    image: ghcr.io/networkservicemesh/ci/cmd-dashboard-backend:latest
```

File: apps/dashboard/dashboard-pod.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      - containerPort: 3001
+    env:
+      - name: SPIFFE_ENDPOINT_SOCKET
+        value: unix:///run/spire/sockets/agent.sock
+    volumeMounts:
+      - name: spire-agent-socket
+        mountPath: /run/spire/sockets
+        readOnly: true
+  - name: dashboard-ui
+    image: ghcr.io/networkservicemesh/ci/cmd-dashboard-ui
```
Comment (denis-tingaikin) [inline apps/dashboard/dashboard-pod.yaml:25]:
```suggestion
    image: ghcr.io/networkservicemesh/ci/cmd-dashboard-ui:latest
```

File: examples/observability/dashboard/README.md (hunk 2 above)
Comment (VitalyGushin) [reply examples/observability/dashboard/README.md]: done

File: examples/observability/dashboard/README.md (hunk 1 above)
Comment (VitalyGushin) [reply examples/observability/dashboard/README.md]: thanks, fixed

File: apps/dashboard/dashboard-pod.yaml (hunk 4 above)
Comment (VitalyGushin) [reply apps/dashboard/dashboard-pod.yaml]: fixed

File: apps/dashboard/dashboard-pod.yaml (hunk 3 above)
Comment (VitalyGushin) [reply apps/dashboard/dashboard-pod.yaml]: fixed

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**